apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-welcome-tour/src/tour-launch.js

```diff
@@ -53,6 +56,7 @@ function LaunchWpcomWelcomeTour() {
 
 		recordTracksEvent( 'calypso_editor_wpcom_tour_open', {
 			is_gutenboarding: window.calypsoifyGutenberg?.isGutenboarding,
+			is_manually_opened: isTourManuallyOpened,
```

Comment:
`isTourManuallyOpened` needs to be added to the list of `useEffect` dependencies.

Reply:
Ah great catch!!!   I was observing some quirky behavior with this earlier but I couldn't consistently reproduce it, I think the missing dependency in useEffect was the root cause.